src/components/EditableLabel/EditableLabel.tsx

```diff
@@ -8,7 +8,11 @@ export interface RootBodyProps {
   style?: CSSProperties;
 }
 
-export default function EditableLabel({ value: initValue = "", onBlur = () => {} }: RootBodyProps): JSX.Element {
+export default function EditableLabel({
+  value: initValue = "",
+  onBlur = () => {},
+  ...props
+}: RootBodyProps): JSX.Element {
```

Comment:
I think that is what I wrote, but is it possible for you to make it a bit more descriptive? I suppose this interface is meant to be used in some places, I don't think this is the proper place to declare. Or renaming it to something more descriptive might help make it understandable.